### PR TITLE
Sync v2-5-stable with v2-5-test to release 2.5.3

### PR DESCRIPTION
**Pull request by pierrejeambrun**
`2.5.3rc1` :tada:

### CODE REVIEW

**Review by potiuk — APPROVED** [review]
LGTM. I just have small doubt about https://github.com/apache/airflow/pull/30285/commits/73fd13f4713e3dca54bc261f07080ca8dcc4b142 -> if it is not needed for something else as "chain of commts" I would remove this one. It is low-risk, but I've seen too many cases where moving or removing an import caused a completely unexpected problem, that I would approach such changes with caution in patchlevel release.

**Comment by pierrejeambrun** [comment]
Yes, right. I just dropped it

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -455,8 +510,8 @@ Significant Changes
 Make ``RotatingFilehandler`` used in ``DagProcessor`` non-caching (#27223)
 """"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-  In case you want to decrease cache memory when ``CONFIG_PROCESSOR_MANAGER_LOGGER=True``, and you have your local settings created before,
-  you can update ``processor_manager_handler`` to use ``airflow.utils.log.non_caching_file_handler.NonCachingRotatingFileHandler`` handler instead of ``logging.RotatingFileHandler``. (#27065)
+In case you want to decrease cache memory when ``CONFIG_PROCESSOR_MANAGER_LOGGER=True``, and you have your local settings created before,
```
Comment (ephraimbuddy) [inline RELEASE_NOTES.rst:513]: Why these changes on previous release notes? I think the indentation here is for a release  cc @jedcunningham

Comment (jedcunningham) [reply RELEASE_NOTES.rst:513]: The indentation impacts the rendering of the releases notes, this is a minor thing that just cleans the release notes up a bit:
[With indentation](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#make-rotatingfilehandler-used-in-dagprocessor-non-caching-27223)
[No indentation](http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/release_notes.html#make-rotatingfilehandler-used-in-dagprocessor-non-caching-27223)

Comment (pierrejeambrun) [reply RELEASE_NOTES.rst:513]: Yes Jed cleaned the Release note to be more consistant.
